components/net_traits/lib.rs

```diff
@@ -91,6 +155,8 @@ pub struct LoadData {
     /// The policy and referring URL for the originator of this request
     pub referrer_policy: Option<ReferrerPolicy>,
     pub referrer_url: Option<Url>,
+    #[ignore_heap_size_of = "Defined in hyper"]
```

Comment:
Yeah, this was an issue i had, when i put derive HeapSizeOf attribute on RequestSource, due to the IpcSender wrapped in it. Do i have to make an impl HeapSizeOf for IpcSender ?